### PR TITLE
New/GUM 60 registration redirect

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,8 +22,18 @@
...
         <intent-filter>
-            <action android:name="android.intent.action.MAIN" />
-            <category android:name="android.intent.category.LAUNCHER" />
+          <action android:name="android.intent.action.MAIN" />
+          <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+        <intent-filter>
+          <action android:name="android.intent.action.MAIN" />
+          <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
```
Comment (AleksandrTermenzhy) [inline android/app/src/main/AndroidManifest.xml]: Why do we need 2 identical `intent-filter`?

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,8 +22,18 @@
...
+        <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+          <data android:scheme="growupmobile" />
```
Comment (AleksandrTermenzhy) [inline android/app/src/main/AndroidManifest.xml]: We also have to handle 'https://bsa-growup.herokuapp.com' kind of links

File: src/navigation/navigation-container/navigation-container.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (AleksandrTermenzhy) [inline src/navigation/navigation-container/navigation-container.tsx]: Let's move it to `src/components`

**Review by AleksandrTermenzhy — APPROVED** [review]

File: src/common/types/navigation/params/root-stack-param.ts [hunk 4]
```diff
@@ -7,6 +7,7 @@ type RootStackParamList = {
   [AppRoute.APP]: NavigatorScreenParams<AppStackParamList>;
   [AuthRoute.SIGN_IN]: undefined;
   [AuthRoute.SIGN_UP]: undefined;
+  [AuthRoute.COMPLETE_REGISTRATION]: undefined;
```
Comment (katyakostikova) [inline src/common/types/navigation/params/root-stack-param.ts]: This screen has token as param
```suggestion
  [AuthRoute.COMPLETE_REGISTRATION]: {
       token: string
  };
```

**Review by katyakostikova — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**